UserManagement/accounts/migrations/0001_initial.py

```diff
@@ -63,11 +63,21 @@ class Migration(migrations.Migration):
                     models.DateTimeField(default=django.utils.timezone.now),
```

Comment:
What timezone are we using ?

Do we need to keep all the times in single zone ?